observability/WFLY-20296-add-support-for-more-opentelemetry-config-options.adoc

```diff
@@ -0,0 +1,154 @@
+||TLS client certificate|`otel.exporter.otlp.client. certificate`|The path to the client certificate on disk|TBD||
+||Client key|`otel.exporter.otlp.client. key`|The path to the key on disk|TBD||
+||Compression|`otel.exporter.otlp. compression`|The compression type to use on OTLP trace, metric, and log requests.|`compression`|null|Only `gzip` supported
+|Traces|Enabled|`otel.traces.exporter`|Are traces enabled?|`traces-enabled`|true|If true, the value is set to `otlp`. If not, the value is set to `none`.
+|Metrics|Enabled|`otel.metrics.exporter`|Are metrics enabled?|`metrics-enabled`|true|If true, the value is set to `otlp`. If not, the value is set to `none`.
+||Export interval|`otel.metrics.export.interval`|The interval, in milliseconds, between the start of two export attempts.|`metrics-export-interval`|60000|
+||Exemplar filter|`otel.metrics.exemplar.filter`|The filter for exemplar sampling.|`exemplar-filter`|`trace_based`|Options are `always_off`, `always_on` or `trace_based`.
+||Metrics cardinality limit|`otel.experimental.metrics. cardinality.limit`|If set, configure cardinality limit; the maximum number of distinct points per metric.|cardinality-limit|2000|
+||Temporality preference|`otel.exporter.otlp.metrics. temporality.preference`|The preferred output aggregation temporality.|`temporality`|`cumulative`|Options include `delta`, `lowmemory`, and `cumulative`.
+||Histogram aggregation|`otel.exporter.otlp. metrics.default. histogram.aggregation`|The preferred default histogram aggregation.|`histogram-aggregation`|`explicit_bucket_histogram`|Options include `base2_exponential_bucket_histogram` and `explicit_bucket_histogram`
+|Logs|Enabled|`otel.logs.exporter`|Are logs enabled?|`logs-enabled`|true|If true, the value is set to `otlp`. If not, the value is set to `none`.
+||Schedule delay|`otel.blrp.schedule.delay`|The interval, in milliseconds, between two consecutive exports.|`logs-export-interval`|1000|
+|===
+* Changes to the management model should be split out based on signal types
+** For ease of use by users, the structure of the updated XML should reflect this grouping of signal-specific attributes via `setAttributeGroup(String)`
+** The shape of the management model should be flat, using a list of attributes, rather than nested resources
+*** Each attribute will be declared on the existing `OpenTelemetrySubsystemRegistrar`. No new child resources should be created.
+*** Attribute groups will be used to group properties by signal type to make the XML file and admin console more user-friendly
+** Shared properties (e.g., the OpenTelemetry Collector endpoint) should be exposed on the root element.
+*** Signal-specific overrides need not be supported
+* All new properties added will have, as defaults, values described in the relevant OpenTelemetry documentation (see the official docs https://opentelemetry.io/docs/languages/java/configuration/[here])
```

Comment:
s/as defaults/as initial defaults/g

Once we set a default, it's *our* default. It's no longer determined based on what some outside party decides. Changing it is a semi-breaking or breaking change, depending on what the attribute configures. (We can change accept that and change the default but we should not promise that we will always do what some outside party said.)